src/mono/netcore/System.Private.CoreLib/src/System/Reflection/Emit/ModuleBuilder.Mono.cs

```diff
@@ -415,7 +415,7 @@ public override Type GetType(string className, bool throwOnError, bool ignoreCas
             if (className == null)
                 throw new ArgumentNullException(nameof(className));
             if (className.Length == 0)
-                throw new ArgumentException("className");
+                throw new ArgumentException(nameof(className));
```

Comment:
The first argument to ArgumentException is a message, not the parameter name (unfortunately).

Reply:
Literally 20 minutes ago I was pointing out on the analyzers issue that we had several instances of this mistake. That's my excuse.

Reply:
Ok :smile: